values-in/strings.xml

```diff
@@ -2501,35 +2501,35 @@ Tekan suatu tombol untuk menetapkan tindakannya.</string>
     <string name="featureselector_doubledincometime">Penghasilan ganda untuk %1$s</string>
     <string name="draft_marker_powermarker_text">Cakupan listrik. Listrik disalurkan menggunakan kabel.</string>
     <string name="draft_marker_waterpipemarker_title">Air</string>
-    <string name="draft_marker_waterpipemarker_text">Cakupan air. Air disalurkan untuk pendudukmu dengan pipa.</string>
-    <string name="stage_files_title">Manajer berkas</string>
+    <string name="draft_marker_waterpipemarker_text">Cakupan air. Air disalurkan untuk pendudukmu menggunakan pipa.</string>
+    <string name="stage_files_title">Manajer file</string>
     <string name="stage_files_importhere">Impor ke sini</string>
-    <string name="stage_files_currentdir">Folder saat ini</string>
+    <string name="stage_files_currentdir">Folder ini</string>
     <string name="stage_files_actions">Aksi</string>
     <string name="stage_files_actionexport">Ekspor</string>
     <string name="stage_files_actionrename">Ganti nama</string>
     <string name="stage_files_actionrename_text">Ganti nama file %1$s:</string>
     <string name="stage_files_renamecollision_title">Kesalahan Penamaan</string>
-    <string name="stage_files_renamecollision_text">File %1$s sudah ada!</string>
+    <string name="stage_files_renamecollision_text">File dengan nama %1$s sudah ada!</string>
     <string name="stage_files_actiondelete">Hapus</string>
     <string name="stage_files_actiondelete_title">Hapus?</string>
     <string name="stage_files_actiondelete_text">Apakah kamu yakin ingin menghapus %1$s? Aksi ini tidak bisa diurungkan!</string>
     <string name="stage_files_settings">Pengaturan</string>
     <string name="stage_files_settings_showhiddenfiles">Tampilkan file yang disembunyikan</string>
     <string name="stage_files_filesize">Ukuran: %1$s</string>
     <string name="stage_files_helptitle">Bantuan</string>
-    <string name="stage_files_helptext">Selamat datang di manajer berkas TheoTown. Di sini kamu bisa mengakses dan mengatur map, region, plugin, dan sebagainya.</string>
-    <string name="unit_sizebytes">%1$,d bytes</string>
+    <string name="stage_files_helptext">Selamat datang di manajer file TheoTown. Di sini kamu bisa mengakses dan mengatur peta, wilayah, plugin, dan sebagainya.</string>
+    <string name="unit_sizebytes">%1$,d byte</string>
```

Comment:
This could be debated, as game will show file size in plural word 'bytes'

Reply:
ok